### PR TITLE
getLatestPrice

### PR DESCRIPTION
**Pull request by solidoracle**

### CODE REVIEW

File: bulla-contracts/src/mappings/BullaFactoring.ts [hunk 1]
```diff
@@ -30,15 +30,19 @@ export function handleInvoiceFunded(event: InvoiceFunded): void {
   InvoiceFundedEvent.fundedAmount = ev.fundedAmount;
   InvoiceFundedEvent.originalCreditor = ev.originalCreditor;
   const original_creditor = getOrCreateUser(ev.originalCreditor);
+  // Update the price history
   const price_per_share = getOrCreatePricePerShare(event);
```
Comment (bengobeil) [inline bulla-contracts/src/mappings/BullaFactoring.ts:34]: Not used?

**Review by bengobeil — APPROVED** [review]
Good call. Let's remove the code pertaining to PriceHistory

File: bulla-contracts/src/mappings/BullaFactoring.ts (hunk 1 above)
Comment (solidoracle) [reply bulla-contracts/src/mappings/BullaFactoring.ts:34]: don't we use that for the pricing of BFT? we need the array of historical prices

Comment (solidoracle) [reply bulla-contracts/src/mappings/BullaFactoring.ts:34]: we save it at the bottom of each function to have it available

Comment (bengobeil) [reply bulla-contracts/src/mappings/BullaFactoring.ts:34]: Doesn't this change replace that mecanism?

**Pull request merged**